### PR TITLE
Chapter describing how to enter the project

### LINKED ISSUE #1732: A chapter on paths to enter The Turing Way
### Summary

* In my talks I highlight many pathways to enter the project, which we can add on the community handbook as a chapter

### What needs to be done?

- [ ] In this issue collect all the different pathways we have designed in the Turing Way
- [ ] Many pathways are described in the acknowledgement chapter so we can focus more on ways to step into the project

### Who can help?

* Anyone from the core community (existing contributors - not suitable for the first time contributors) interested in writing this with me

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #1732 

* This is a chapter providing overview of paths to enter the project, community resources we have and how to stay involves

### List of changes proposed in this PR (pull-request)

* [x] Landing page with a resource list
* [ ] Motivations and ongoing community projects
* [ ] Workflow and process

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@Lovkush-A issue #1732 is being written here. Welcome to The Turing Way. Please start by reading this contribution guideline: https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md

If you are new to GitHub, please read this chapter to get a bit familiar with the system: https://the-turing-way.netlify.app/collaboration/github-novice.html

We have described the review process here: https://the-turing-way.netlify.app/collaboration/github/github-maintainers.html.

if you would like to start small, you can also take a stab at this PR that has a few small edits that you can look through (and test if you can) and comment under the issue if anything should be changed or everything looks ok: https://github.com/alan-turing-institute/the-turing-way/pull/1745

File: book/website/_toc.yml [hunk 1]
```diff
@@ -365,7 +367,7 @@
       file: community-handbook/consistency/consistency-structure
     - title: Language
       file: community-handbook/consistency/consistency-language
-  - title: Contributing and Developing Chapters - Templates and Workflow
+  - title: Chapter Templates and Contribution Workflow
```
Comment (Lovkush-A) [inline book/website/_toc.yml:370]: I think this edit should be in a separate pull request/issue, but I am not sure.

File: book/website/community-handbook/contributing.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
 (ch-contributing)=
-# Contributing and Developing Chapters - Templates and Workflow
+# Chapter Templates and Contribution Workflow
 
 As _The Turing Way_ project grows and evolves, we actively welcome and encourage continuous contributions from community members in the form of new chapters and case studies. 
 However, these new contributions need to be consistent with the overall theme, purpose, format, and style of the book's existing content.
 
 Since _The Turing Way_ is written asynchronously by multiple authors around the world, we created a series of templates to guide different kinds of content development while maintaining a consistent format of the book.
 
+```{figure} ../figures/templates.png
```
Comment (Lovkush-A) [inline book/website/community-handbook/contributing.md:9]: I think this edit should be in a separate pull request/issue, but I am not sure.

File: book/website/community-handbook/enter-the-project.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+(ch-enter-the-project)=
+
+# How to enter *The Turing Way*
+
+*The Turing Way* is an open source project that involves a diverse community of researchers in making data science reproducible, ethical and inclusive for everyone.
```
Comment (Lovkush-A) [inline book/website/community-handbook/enter-the-project.md:5]: * I think this introduction could be shorter and punchier. E.g. I do not think it is necessary to explain what the Turing Way in this chapter.
* I think there should be a sub-heading to indicate this is an introduction

File: book/website/community-handbook/enter-the-project.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+Since 2019, the project has grown into an international community of people with diverse backgrounds, research domains, expertise, knowledge and lived-experiences. 
+Simultaneously, multiple pathways have evolved that interested members can take to engage in the project.
+Though most of these pathways have been mentioned in various chapters, we have also noticed that often it is hard for people to enter an ongoing project when they don't know what their role in the community would look like.
+
+(ch-overview)=
+## Overview of Resources You Can Use
+
+First, we provide an overview of different platforms, community-led efforts, possible roles and opportunities for collaboration in the community.
+
+In the next subchapters, you will find descriptions of different resources mentioned in the overview table.
```
Comment (Lovkush-A) [inline book/website/community-handbook/enter-the-project.md:22]: * Turn this paragraph into a bulleted list

File: book/website/community-handbook/enter-the-project.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+Then we describe other ongoing efforts in the project that you can use to join and ideas for the kind of initiatives you can lead while collaborating with others in continuing your journey within *The Turing Way*.
+Finally, we share some personal motivations of our current members for participating in this project and aspects of their work they find meaningful and beneficial.
+
+## 1. Get to Know the Project
```
Comment (Lovkush-A) [inline book/website/community-handbook/enter-the-project.md:26]: Should this be a level three subheading? (i.e. ###). It looks like this is a part of the `overview of resources you can use` section.

**Review by Lovkush-A — COMMENTED** [review]
Overall, I like it! Especially the tables which lay out clearly the various helpful resources available.

Some comments:
* There is significant repetition between this and the contributing guidelines. Is this intentional? I personally find it confusing; now there are two places that I would consider going to if I need to get an answer to something.
* There are various things that I would re-word or re-organise. E.g. learning about community feels like it should be in the first table.
* There are few little typos. (Is there a way I can make changes to this?)
* The number of different options is a bit overwhelming. Here are a couple of ideas:
* * Have the first page be some absolute minimum and easy starting point about how to enter, and then . E.g. it could be: 1. read chapters x,y,z of book; 2. say hello on slack; 3. do some minimal PR (find a typo, adding oneself to a list of contributors); 4. find an issue that sparks your interest and post a message to say you want to help; 5. join us in this upcoming meetup.
* * Have a basic 'Make your first contribution to Turing tutorial', in a super-hand holdy way, for people totally new to open-source.

**Comment by malvikasharan** [comment]
Hey @Lovkush-A. Thanks for the feedback. The repetition is intentional as folks who are not contributors or don't intend to be contributors right away, may not go and read the contribution guideline in details. Furthermore, there are a lot of norms that have evolved organically which have not been captured comprehensively. With this chapter, I aim to address those.

Since I will not be able to address these before mid-March, I welcome you to take over the editing if you'd like. Would that be something you'd be interested in?

**Comment by Lovkush-A** [comment]
@malvikasharan Yes, I would be happy to edit this. I think it would be good to have a quick chat so I know the following:
1) The ultimate vision/goal of the chapter, and hence an idea of overall structure / list of subchapter names for the chapter.
1a) Who are the intended beneficiaries? Do we need separate advice for different kinds of intended beneficiaries?
2) Where I can find the information that would fill in all the sub chapters.

**Comment by malvikasharan** [comment]
This chapter is for anyone learning about The Turing Way and/or sharing information about The Turing Way. Based on our interaction so far, I highly encourage you to read the community handbook before editing this chapter as it lays out the foundation for what this project and community stand for. We don't aim to write a traditional academic/tech chapter as may have noticed. We prioritise accessibility over perfection/polished. My intention is to make it as easy as possible for people to read and learn, rather than create just another resource that is already available in excess on the internet.

Since you are new to the project, I suggest you take smaller tasks listed in issues marked as 'first good issue' before diving into niche chapters like this one, which will require a lot of time from my side. There are many previous PRs that can also use some review and help.
For example, this chapter on outreach: https://github.com/alan-turing-institute/the-turing-way/pull/1709.

**Comment by Lovkush-A** [comment]
Thanks for explanation! I will read the community guidelines and also work on smaller issues/reviews, instead of a big task like this chapter.

One small thing, I think that 'perfection/polished' is part of accessibility and making things easy to read and learn. But maybe I have misunderstood - I suppose this is why I should read the community guidelines!

**Comment by malvikasharan** [comment]
Here is the community handbook: https://the-turing-way.netlify.app/community-handbook/community-handbook.html

**Comment by malvikasharan** [comment]
This document will be added to the foreword that is described here #2033.
